app/models/user.rb

```diff
@@ -0,0 +1,15 @@
+class User < ActiveRecord::Base
+	before_save { self.email = email.downcase }
+	VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+
+	validates_presence_of :full_name
+	validates :email, presence: true, length: { maximum: 255 },
+		format: { with: VALID_EMAIL_REGEX}, uniqueness: { case_sensitive: false }
```

Comment:
This should be formatted in a different way.

```ruby
validates :email, presence: true,
                  length: { maximum: 255 },
                  format: { with: VALID_EMAIL_REGEX },
                  uniqueness: { case_sensitive: false }
```
this reads better as it is more clear. also you missed space after `VALID_EMAIL_REGEX`.